pq-crypto/EVP_kem.h

```diff
@@ -12,12 +12,11 @@
 #include <stdint.h>
 #include "sike_r3/sike_internal.h"
 
-// -----------------------------------------------------------------------------
+/// -----------------------------------------------------------------------------
```

Comment:
Can you please make sure that all lines are at most 80 characters long (for the code/comments you are adding, not for SIKE's code)?